azure-pipelines.yml

```
@@ -130,6 +130,21 @@ jobs:
       condition: and(succeeded(), not(eq(variables['skip-github'], 'TRUE')))
     - bash: |
         set -euo pipefail
+        if [ -d sdk ]; then
+          cd sdk
+        fi
```

```suggestion
        cd sdk
```



Do we expect to publish from a branch that doesn't have the SDK directory? The last branch that doesn't have the sdk subdirectory is 2.7.x, and I don't think we're planning any releases from that branch, so we can just change directory and fail when the directory doesn't exist.